src/main/java/com/kuba6000/mobsinfo/loader/MobRecipeLoader.java

```diff
@@ -223,13 +225,13 @@ public void newRound() {
         public boolean nextRound() {
             walkCounter = 0;
             chance = 1d;
-            while (nexts.size() > 0 && nexts.get(nexts.size() - 1)
+            while (!nexts.isEmpty() && nexts.get(nexts.size() - 1)
                 .next()) nexts.remove(nexts.size() - 1);
-            return nexts.size() > 0;
+            return !nexts.isEmpty();
         }
     }
 
-    private static class dropinstance {
+    public static class dropinstance {
```

Comment:
Why is the access modifier changed?

Reply:
dropinstance is used as an argument in and returned by a public method. Technically it's "fine," but it can lead to problems if someone actually tries to use the public method somewhere else. I could make the methods using the class private instead, as long as it's not used somewhere else.